.github/workflows/build.yml

```diff
@@ -30,99 +30,99 @@ jobs:
       - name: Setup .NET Core
         uses: actions/setup-dotnet@v2
         with:
-          dotnet-version: 6.0.x
+          dotnet-version: 8.0.x
 
       - name: Build
-        run: dotnet publish -c ${{ matrix.configuration }} -o artifacts
+        run: dotnet publish -c ${{ matrix.configuration }} --property:PublishDir=${{env.DOTNET_ROOT}}/artifacts
```

Comment:
writing to DOTNET_ROOT seems horribly wrong to me

Reply:
Would you prefer home folder?
Or maybe /tmp , it should work and make sense usage wise

Reply:
Nvm, /tmp probably won't work since it also runs on windows, i'll look for a saner var to use

Reply:
RUNNER_TEMP should do the trick 
https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

Reply:
Seems like i misunderstood the docs, `runner.temp` works